pkg/ssl/sign_test.go

```diff
@@ -67,22 +68,22 @@ func (n nilsigner) Sign(data []byte) ([]byte, string, error) {
 	return data, "nil", nil
 }
 
-func (n nilsigner) Verify(keyID string, data, sig []byte) (bool, error) {
+func (n nilsigner) Verify(keyID string, data, sig []byte) error {
 
 	if keyID == "nil" {
 		if len(data) != len(sig) {
-			return false, nil
+			return errors.New("invalid length")
```

Comment:
It might make sense to make error types out of these static error strings if we use them more than once.
For example:

`var ErrInvalidPayloadLength = errors.New("invalid payload length")`

For example like this: https://github.com/in-toto/in-toto-golang/blob/master/pkg/ssl/sign.go#L15L23

This way we can add more more documentation to errors (if needed). More Information can be found here: https://blog.golang.org/go1.13-errors

Reply:
All set! These are tests so I left the errs private rather than exposing them outside the package.